kedro/framework/cli/hooks/manager.py

```diff
@@ -17,6 +19,8 @@ def get_cli_hook_manager():
     global _cli_hook_manager
     if _cli_hook_manager is None:
         _cli_hook_manager = CLIHooksManager()
+    _cli_hook_manager.trace.root.setwriter(logger.debug)
+    _cli_hook_manager.enable_tracing()
     return _cli_hook_manager
```

Comment:
How would you be able to see these `tracing` log?

Reply:
By increasing the logging level to `DEBUG`:

```
> cat /tmp/test-kedro-iris/logging.yml
version: 1

disable_existing_loggers: False

handlers:
  rich:
    class: rich.logging.RichHandler

loggers:
  kedro:
    level: DEBUG

root:
  handlers: [rich]
> KEDRO_LOGGING_CONFIG=/tmp/test-kedro-iris/logging.yml kedro run 
```

(although, beware - `KEDRO_LOGGING_CONFIG` gets overridden if `conf/base/logging.yml` exists)